utils/pciutils/Makefile

```diff
@@ -40,7 +40,7 @@ endef
 define Package/pciutils/postinst
 #!/bin/sh
 [ -z "$${IPKG_INSTROOT}" ] || \
-(cd $${PKG_ROOT}/usr/share; $${PKG_ROOT}/usr/sbin/update-pciids; rm pci.ids.gz.old)
+(cd $${PKG_ROOT}/usr/share; $${PKG_ROOT}/usr/sbin/update-pciids; rm -f pci.ids.gz.old)
```

Comment:
why is this run in ```postinst``` on the build host? shouldn't this be run in ```prepare``` or ```build``` step ??

Reply:
Possibly. I'm not super familiar with Debian packaging (I'm an RPM user myself).

Reply:
also this prevent reproducible builds
maybe we should fetch the pci.ids database from https://github.com/pciutils/pciids instead of using update-pciids scripts
@Noltari ?